### PR TITLE
Bump Plutarch/LPE; Some small clean up

### PR DESCRIPTION
**Pull request by chfanghr**

### CODE REVIEW

**Review by peter-mlabs — COMMENTED** [review]
This all looks good to me except for the version bump. 

Through the reformatting, I'm noticing a lot of types like

```
 forall k.
  forall (to :: k) (from :: k) (s :: S).
  Term s (PExchangeRate from to :--> PDiscrete from :--> PDiscrete to)
```

Can that first `forall k.` get a kind signature? It seems to still compile with `k :: Type`, but I'm not sure if this is incorrect?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
+## 1.3.0 -- 2022-09-01
```
[the remark this answers is not in the document]
Comment (chfanghr) [reply CHANGELOG.md:5]: There was already `1.2 code when I started making changes.

**Comment by kozross** [comment]
@peter-mlabs Yes, `Type` is correct here, because `Type :: Type` is a GHC axiom. However, is there a reason why those functions need to be poly-kinded? I can't imagine under what circumstances `from` and/or `to` will have kinds that aren't `S -> Type`.

**Comment by peter-mlabs** [comment]
@kozross , I think you're correct there -- I think we can just make those `S -> Type`

**Review by SeungheonOh — APPROVED** [review]
LGTM, Just want to make sure: this PR is formatting + `Discrete` and only that, am I correct?

**Comment by chfanghr** [comment]
Yep!

**Comment by SeungheonOh** [comment]
Perfect!

**Pull request merged**